testing/sbt/APKBUILD

```diff
@@ -0,0 +1,45 @@
+# Contributor: Ed Robinson <ed@reevoo.com>
+# Maintainer: Ed Robinson <ed@reevoo.com>
+pkgname=sbt
+pkgver=0.13.13
+pkgrel=0
+pkgdesc="The interactive build tool, sbt is a build tool for Scala, Java, and more."
+url="http://www.scala-sbt.org/"
+arch="noarch"
+license="BSD-3"
+depends="openjdk8-jre bash"
+source="https://dl.bintray.com/$pkgname/native-packages/$pkgname/$pkgver/$pkgname-$pkgver.tgz
+global-sbtopts.patch"
+
+builddir="$srcdir/$pkgname-launcher-packaging-$pkgver"
+_basedir="/usr/share/java/$pkgname"
+
+prepare() {
+	default_prepare || return 1
+	cd "$builddir"
+
+	# Remove windows specific files
```

Comment:
why are we removing these files?  you say they are windows specific, but there's no need to remove them just because of that.